### DISCUSSION TITLE
ChatGPT 4

### DISCUSSION THREAD

**Discussion by DonEugraf** [Yue-Yang/ChatGPT-Siri#42] [General]
Hi,
If I had this working with chatGPT 3.5 does it now run automatically with 4.0 or do I need to do anything else?

**Comment by XOKP** [comment]
An invitation is required to access GPT-4.

**Reply by Yue-Yang** [reply]
GPT-4 is currently in a limited beta and only accessible to those who have been granted access. Please [join the waitlist](https://openai.com/waitlist/gpt-4) to get access when capacity is available.

**Reply by trixlme** [reply]
If using a plus account I have access to gpt4, but how can this link to this Siri integration?

**Reply by XOKP** [reply]
The API access to GPT-4 is not related to your Plus account. Having Plus or not, does not affect whether or not you have API access to GPT-4. To gain access, you must apply with the waitlist.

**Reply by rpc31** [reply]
Everything works well with GPT 3.5. I have OpenAI PLUS with access to ChatGTP 4 via the web. How do I know if I have access to chatGPT 4 via the API?

**Reply by XOKP** [reply]
Again, API access has nothing to do with the Plus account.. You must apply through the waitlist for GPT-4.
